amazon-sns/4.7/modules/ROOT/pages/amazon-sns-connector-reference.adoc

```diff
@@ -1210,7 +1212,7 @@ The Subscribe action prepares to subscribe an endpoint by sending the endpoint a
 ** SMS
 ** SQS
 ** APPLICATION
-** LAMBDA |  The protocol you want to use. |  | x
+** LAMBDA |  The protocol you want to use. If it is a FIFO topic, then only Amazon SQS FIFO queues can be subscribers. |  | x
```

Comment:
Not sure what "it" refers to here. The endpoint?

Reply:
will verify